.circleci/config.yml

```diff
@@ -398,7 +398,7 @@ jobs:
             ccache -sz -M 5Gi
             source /opt/rh/gcc-toolset-9/enable
             make clean
-            make debug EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_TESTING=OFF -DVELOX_ENABLE_SUBSTRAIT=ON" NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=16
+            make debug EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_SUBSTRAIT=ON" NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=16
```

Comment:
Looks like this action will generate test executables into the `_build/debug` folder. But at below, the tests in release folder are triggered with `cd _build/release && ctest -j 16 -VV --output-on-failure`.

Reply:
> Looks like this action will generate test executables into the `_build/debug` folder. But at below, the tests in release folder are triggered with `cd _build/release && ctest -j 16 -VV --output-on-failure`.

You are correct. Thank you for your review.

Reply:
If I am not mistaken tests can be generated for make release also..

Reply:
> If I am not mistaken tests can be generated for make release also..

I changed "debug" -> "release" now.